portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,60 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+html{
+    height: 100%;
+    width: 100%;
 }
+ 
+body{
+    height: 100%;
+    margin: 0;
+    background: linear-gradient(45deg,
+      rgba(20, 18, 19, 0.9) 30%,
+      rgba(0, 94, 105, 0.9) 100%),
+    url(Images/github_pattern.svg);
+    background-repeat: no-repeat;
+    background-attachment: fixed;
```

Comment:
Declarations should be in alphabetical order. Looks like the other blocks are already sorted so just need to fix this one :)
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Done!